pkg/packetbrokeragent/translation.go

```diff
@@ -23,12 +23,12 @@ import (
 
 	packetbroker "go.packetbroker.org/api/v3"
 	"go.thethings.network/lorawan-stack/v3/pkg/band"
-	"go.thethings.network/lorawan-stack/v3/pkg/cluster"
 	"go.thethings.network/lorawan-stack/v3/pkg/encoding/lorawan"
 	"go.thethings.network/lorawan-stack/v3/pkg/errors"
 	"go.thethings.network/lorawan-stack/v3/pkg/events"
 	"go.thethings.network/lorawan-stack/v3/pkg/frequencyplans"
 	"go.thethings.network/lorawan-stack/v3/pkg/log"
+	stdpb "go.thethings.network/lorawan-stack/v3/pkg/packetbroker"
```

Comment:
In case `stdpb` is too controversial, better naming suggestions are open.

Reply:
Hmm maybe `pkgpb` or even `pkgpacketbroker`.